### ISSUE TITLE
Entity name is not visible

### ISSUE THREAD

**Issue by freekode** [bbbenji/synthwave-hass#32]
I have such problem in Developer Tools.
HA version is 0.109.2
<img width="348" alt="Screenshot 2020-05-11 at 15 23 26" src="https://user-images.githubusercontent.com/940893/81566584-5e540500-939b-11ea-9687-ffc836640f1d.png">

**Comment by Morphy99** [comment]
Same here

**Comment by bbbenji** [comment]
Are you guys running: https://github.com/bbbenji/synthwave-hass-extras?

**Comment by freekode** [comment]
No, It installed with HACS
<img width="379" alt="Screenshot 2020-05-18 at 16 43 33" src="https://user-images.githubusercontent.com/940893/82226381-bbfdc980-9926-11ea-841d-2837c41ef374.png">

**Comment by bbbenji** [comment]
Please install the extras package, contains some workarounds that can't be
themed otherwise. It is also available through HACS. Let me know if  it
resolves your problem so I can close this issue

**Comment by Morphy99** [comment]
> Are you guys running: https://github.com/bbbenji/synthwave-hass-extras?

Yep I am. 0.2.3

**Comment by bbbenji** [comment]
Sometimes you have to visit the main Lovelace page in order to load the
JavaScript. Then the developer tools theming take effect.

**Comment by Morphy99** [comment]
I did edit a file manually a few weeks ago as something wasn't showing correctly, I changed a colour I think. Can't for the life of me remember which and I assumed if I reinstalled the theme it would overwrite those changes. Unless I edited a lovelace file...

**Comment by Morphy99** [comment]
Hold up, this isn't affecting my Developer Tools section. It's when I click on related in entity view 
![image](https://user-images.githubusercontent.com/5853979/82228647-4c391000-9921-11ea-8188-d4dd1fadef2a.png)

If someone else could confirm if this is an issue for them I'll stop hunting for those changes I made!

**Comment by WesSec** [comment]
I have this issue as well (SOLVED, SEE BELOW)
![image](https://user-images.githubusercontent.com/18014192/82242102-95975880-993d-11ea-96c2-3b4e25a4a7a3.png)

my vacuum-card's toolbar also has some color issues. I think they're related. I did not change anything in the theme files and have the extra's installed

EDIT:

When i disable the following option:
![image](https://user-images.githubusercontent.com/18014192/82242492-371eaa00-993e-11ea-8968-81f5d779cdef.png)
the icons do get a normal color :)

![image](https://user-images.githubusercontent.com/18014192/82242506-40a81200-993e-11ea-9c84-f267c0b30d41.png)

EDIT 2: it was indeed the extra's not being loaded properly, a reinstall fixed the issues!

**Comment by Morphy99** [comment]
@wessel145 I just tried reinstalling the extra's but no luck. Can you check to see if "related" tab of a device is showing like mine please?

**Comment by Morphy99** [comment]
I found the edit I made [here](https://github.com/bbbenji/synthwave-hass/issues/24) but this has been merged so I'm guessing not the problem. I've also noticed the new devices view has this problem: 
![image](https://user-images.githubusercontent.com/5853979/82764841-dc87c100-9e09-11ea-99cd-9dc1d718d59c.png)

**Comment by martinjuhasz** [comment]
I had the same problem with the devices view and the states page. 
Was fixed by installing synthwave-hass-extras. Would be cool if this also could be installed by HACS.

**Comment by Morphy99** [comment]
I have the extras installed. I've just noticed the above device view is working now. However a few things are still not showing correctly. Related tab in entity settings:
![Screenshot_20200616-180108_Home Assistant](https://user-images.githubusercontent.com/5853979/84805231-d8f7ec00-affb-11ea-9da6-3a148579f954.jpg)
Also, under configuration, the selected tab name text is not visible.
Again, under Devices view inside a device the + icon isn't visible either.

**Comment by Morphy99** [comment]
Ok so upon further testing I've noticed the issue above with the "related" tab is present on all devices/browsers. 
It seems there is another issue with synthwave extras not being loaded after HA restart or browser restart maybe. This only seems to affect my Windows x64 Chrome and not HA Android App. I've tried clearing data on it but the only thing that works is switching to another theme clicking around menus then switching back to syncthwave and again clicking around again. Weird. Have a look at [this](https://photos.app.goo.gl/zCbpJNXPPBJz1HD46) video as well, it shows the H in home assistant changing size between tabs
